docs/running-amplication-platform/connect-server-to-github.md

```diff
@@ -33,32 +33,29 @@ When running a local Amplication server you first need to configure the server t
      - **Webhooks** - `Access: Read and Write`
 3. Click `Create GitHub app`
 4. Click `Generate a new client secret` and copy the resulting secret
-5. Click `Generate a private key`, download, open the generated certificate, add `\n` at the end of each line except the last line and join all lines to obtain a single line string
-6. Under the "Generate a private key" section, click "Generate private key" to download a PEM file containing your app's private key
-7. Note the App ID shown on the "General" tab of your app's settings page
-8. On your local machine, set the GITHUB_APP_APP_ID and GITHUB_APP_PRIVATE_KEY environment variables to the App ID and the contents of the PEM file, respectively
-9. Once you have set up these credentials, you should be able to complete the build process locally using git pull.
+5. Under the "Generate a private key" section, click `Generate a private key`, download the PEM file containing your app's private key, open the generated certificate, add `\n` at the end of each line except the last line and join all lines to obtain a single line string
+6. Note the App ID shown on the "General" tab of your app's settings page
 
 ## Step 2: Configure Amplication server to work with the new GitHub app
 
-10. Clone [/packages/amplication-server/.env](https://github.com/amplication/amplication/blob/master/packages/amplication-server/.env) into `/packages/amplication-server/.env.local`
-11. Update `/packages/amplication-server/.env.local` with the following variables
+1. Clone [/packages/amplication-server/.env](https://github.com/amplication/amplication/blob/master/packages/amplication-server/.env) into `/packages/amplication-server/.env.local`
+2. Update `/packages/amplication-server/.env.local` with the following variables
 
    ```sh title=".env.local"
    # GitHub App (Git sync)
-   GITHUB_APP_APP_ID="replace with the github App ID"
+   GITHUB_APP_APP_ID="replace with the github App ID as step 1.6"
    GITHUB_APP_CLIENT_ID="replace with the github app Client ID"
    GITHUB_APP_CLIENT_SECRET="replace with secret created as step 1.4"
-   GITHUB_APP_PRIVATE_KEY=-----BEGIN RSA PRIVATE KEY-----\nContent of the key created in step 1.5\n-----END RSA PRIVATE KEY-----
+   GITHUB_APP_PRIVATE_KEY="replace with content of the key created in step 1.5"
```

Comment:
Is Github now generating the key already with this prefix/postfix?

Reply:
yep, as any PEM certificate